README.md

```diff
@@ -70,7 +70,7 @@ kubectl config use-context <name>
 Example first deployment to a local cluster (from the project root directory):
 ```  
  helm upgrade --install palisade . \
-  --set global.persistence.classpathJars.local.hostPath=$(pwd),global.persistence.dataStores.palisade-data-store.local.hostPath=$(pwd),global.persistence.kafka.local.hostPath=$(pwd),global.persistence.redisMaster.local.hostPath=$(pwd),global.persistence.redisSlave.local.hostPath=$(pwd),global.persistence.zookeeper.local.hostPath=$(pwd),traefik.install=true,kafka.install=true,redis.install=true,global.hosting=local,redis-cluster.install=false --timeout=200s
+  --set global.persistence.statefulset.pv.enabled=true,global.persistence.classpathJars.local.hostPath=$(pwd),global.persistence.dataStores.palisade-data-store.local.hostPath=$(pwd),global.persistence.kafka.local.hostPath=$(pwd),global.persistence.redisMaster.local.hostPath=$(pwd),global.persistence.redisSlave.local.hostPath=$(pwd),global.persistence.zookeeper.local.hostPath=$(pwd),traefik.install=true,kafka.install=true,redis.install=true,global.hosting=local,redis-cluster.install=false --timeout=200s
```

Comment:
I know it's picky, but should this be camel case i.e. statefulSet like other properties?

Reply:
Good eye.  I'll fix that

Reply:
I disagree, this is a yaml value parameter - keep this lowercase

Reply:
OK, leave it in whatever state you have it in now and we'll raise a ticket to rationalize all of these values.yaml parameters on another ticket.

Reply:
PAL-798 has been created for this issue